src/main/java/no/nav/tag/kontaktskjema/KontaktskjemaController.java

```diff
@@ -35,6 +32,9 @@ public ResponseEntity meldInteresse(
         }
         try {
             kontaktskjema.setOpprettet(LocalDateTime.now());
+            if(repository.findAllNewerThan(LocalDateTime.now().minusMinutes(10)).size() >= 10) {
+                return ResponseEntity.status(HttpStatus.TOO_MANY_REQUESTS).build();
```

Comment:
Her returnerer jeg status umiddelbart i stedet for å kaste en Exception som blir fanget opp lenger nede. Vet ikke hva som er mest ønskelig, men det er iallfall greit å returnere noe annet enn http 500 i dette tilfellet.

Reply:
Enig i det. Den wrappingen med try-catch er uansett ikke så uheldig, burde endre på det på sikt